docs/content/latest/cdc/cdc_documentation_v1.md

```diff
@@ -0,0 +1,187 @@
+  
+### Process Architecture
+
+### CDC Streams
+
+  Streams are the YugabyteDB endpoints for fetching DB changes by applications, processes and systems. Streams can be enabled or disabled [on a per table basis]. Every change to a database table (for which the data is being streamed) is emitted as a record to the stream, which is then propagated further for consumption by applications, in our case to Debezium and then ultimately to Kafka.
+
+  #### DB Stream
+  In order to facilitate the streaming of data, we have to create a DB Stream, this stream is created on the database level and can be used to access the data out of all the tables under a particular database.
+  
+### Consistency Semantics
+  * #### Per-Tablet Ordered Delivery Guarantee
+    All changes for a row (or rows in the same tablet) will be received in the order in which they happened. However, due to the distributed nature of the problem, there is no guarantee of the order across tablets.
+  * #### At least Once Delivery
+    Updates for rows will be streamed at least once. This can happen in the case of Kafka Connect Node failure. If the Kafka Connect Node pushes the records to Kafka and crashes before committing the offset, on the restart, it will again get the same set of records.
+  * #### No Gaps in Change Stream
+    Note that once you have received a change for a row for some timestamp t, you will not receive a previously unseen change for that row at a lower timestamp. Therefore, there is a guarantee at all times that receiving any change implies all older changes have been received for a row.
+
+### Performance Impact
+
+  The change records for CDC are read from the WAL. CDC module maintains checkpoint internally for each of the stream-id and garbage collects the WAL entries if those have been streamed to cdc clients. <br/>
+
+  In case CDC is lagging or away for some time, the disk usage may grow and may cause YugabyteDB cluster instability. To avoid a scenario like this if a stream is inactive for a configured amount of time we garbage collect the WAL. This is configurable by a GFLAG.
+
+### Prerequisites/Consideration
+
+  * You should be using YugabyteDB version <version-number> or higher
+  * You cannot stream data out of system tables
+  * Yugabyte cluster should be up and running, for details see YugabyteDB Quick Start
+  * There should be at least one primary key on the table you want to stream the changes from
+  * You cannot create a stream on a table which doesn’t exist. For example, if you create a DB stream on the database and after that create a new table in that database, you won’t be able to stream data out of that table. A simple workaround is to create a new DB Stream ID and use it to stream data
+
+### yb-admin commands for Change Data Capture
```

Comment:
These should go in the yb-admin reference: docs/content/latest/admin/yb-admin.md